pkg/network/render.go

```diff
@@ -338,13 +338,16 @@ func isNetworkChangeSafe(prev, next *operv1.NetworkSpec, infraRes *bootstrap.Inf
 	case len(prev.ServiceNetwork) > len(next.ServiceNetwork):
 		// Going from dual to single
 		dualStack, singleStack = prev, next
+	case !reflect.DeepEqual(prev.ServiceNetwork, next.ServiceNetwork):
+		return errors.Errorf("unsupported change to ServiceNetwork")
+	case validateCidrExpansion(prev.ClusterNetwork, next.ClusterNetwork) &&
+		reflect.DeepEqual(next.DefaultNetwork.Type, operv1.NetworkTypeOVNKubernetes):
```

Comment:
`reflect.DeepEqual(next.DefaultNetwork.Type, operv1.NetworkTypeOVNKubernetes)` should be a part of the function we call to verify whether the clusternetwork change is safe

Reply:
sounds good. done.